xblock/exceptions.py

```diff
@@ -21,7 +21,7 @@ class XBlockSaveError(Exception):
     """
     Raised to indicate an error in saving an XBlock
     """
-    def __init__(self, saved_fields, dirty_fields):
+    def __init__(self, saved_fields, dirty_fields, message=None, *args):
```

Comment:
I used `*args` if user wanted to pass any other arguments. basically allow users initialize misc.
When i saw the signature of `Exception` (super class of XBlockSaveError) its `__init__` signature was accepting `*args` and `**kwargs`
like this

`def __init__(self, *args, **kwargs)`

We can either add or remove both; which makes more sense. Thoughts?